voter-dapp/src/ResolvedRequests.js

```diff
@@ -0,0 +1,85 @@
+import React, { useMemo } from "react";
+import { drizzleReactHooks } from "drizzle-react";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Typography from "@material-ui/core/Typography";
+
+import { formatDate } from "./common/FormattingUtils.js";
+
+function ResolvedRequests() {
+  const { drizzle, useCacheCall, useCacheEvents } = drizzleReactHooks.useDrizzle();
+  const { web3 } = drizzle;
+
+  const currentRoundId = useCacheCall("Voting", "getCurrentRoundId");
+
+  const { account } = drizzleReactHooks.useDrizzleState(drizzleState => ({
+    account: drizzleState.accounts[0]
+  }));
+
+  const resolvedEvents =
+    useCacheEvents(
+      "Voting",
+      "PriceResolved",
+      useMemo(() => ({ filter: { resolutionRoundId: currentRoundId - 1 }, fromBlock: 0 }), [currentRoundId])
```

Comment:
What happens when currentRoundId hasn't been returned yet?

Reply:
Nice catch. Assuming that it's impossible to conditionally call `getPastEvents`, I filter with a `resolutionRoundId` that'll always return an empty set.